packages/braid-design-system/src/lib/components/Tabs/Tabs.css.ts

```diff
@@ -50,12 +50,7 @@ export const marginAuto = style({
 });
 
 export const tabFocusRing = style({
-  margin: vars.borderWidth.large,
-  selectors: {
-    [`${tab}:focus &`]: {
-      opacity: 1,
-    },
-  },
+  outlineOffset: calc.negate(vars.borderWidth.large),
```

Comment:
This should use the `focusRingSize` to prevent clipping.
```suggestion
  outlineOffset: calc.negate(vars.focusRingSize),
```